pkg/rust/cargo/parse_test.go

```diff
@@ -37,27 +72,40 @@ func TestParse(t *testing.T) {
 			f, err := os.Open(v.file)
 			require.NoError(t, err)
 
-			got, _, err := NewParser().Parse(f)
+			libs, deps, err := NewParser().Parse(f)
 
-			require.NoError(t, err)
+			if !v.wantErr(t, err, fmt.Sprintf("Parse(%v)", v.file)) {
+				return
+			}
+
+			if err != nil {
+				return
+			}
+
+			sortLibs(libs)
+			sortLibs(v.wantLibs)
+
+			sortDeps(deps)
+			sortDeps(v.wantDeps)
 
-			sort.Slice(got, func(i, j int) bool {
-				ret := strings.Compare(got[i].Name, got[j].Name)
-				if ret == 0 {
-					return got[i].Version < got[j].Version
-				}
-				return ret < 0
-			})
-
-			sort.Slice(v.want, func(i, j int) bool {
-				ret := strings.Compare(v.want[i].Name, v.want[j].Name)
-				if ret == 0 {
-					return v.want[i].Version < v.want[j].Version
-				}
-				return ret < 0
-			})
-
-			assert.Equal(t, v.want, got)
+			assert.Equalf(t, v.wantLibs, libs, "Parse libraries(%v)", v.file)
+			assert.Equalf(t, v.wantDeps, deps, "Parse dependencies(%v)", v.file)
 		})
 	}
 }
+
+func sortLibs(libs []types.Library) {
+	sort.Slice(libs, func(i, j int) bool {
+		ret := strings.Compare(libs[i].Name, libs[j].Name)
+		if ret == 0 {
+			return libs[i].Version < libs[j].Version
+		}
+		return ret < 0
+	})
```

Comment:
```suggestion
	sort.Slice(libs, func(i, j int) bool {
		return strings.Compare(libs[i].ID, libs[j].ID) < 0
	})
```
looks like we can just use `ID`.